.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+go:
+  - 1.10.x
+
+before_install:                                                                 
+  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh  
+  - dep ensure
```

Comment:
Is this needed? Isn't `dep ensure` run in via the `Makefile`?

Reply:
Currently `dep ensure` runs in `make build`, so we need it for `make test` to work.